husarion_ugv_hardware_interfaces/README.md

```diff
@@ -1,4 +1,4 @@
-# panther_hardware_interfaces
+# husarion_ugv_hardware_interfaces
 
 Package that implements SystemInterface from ros2_control for Panther.
```

Comment:
I changes every where "Husarion UGV robots" to "Husarion UGV", because when we expand the shortcut, we get the Husarion unmanned ground **vehicle robot**. So vehicle seems enought.